### DISCUSSION TITLE
How do I work with commands like "Claims.PreventTheft: true"?

### DISCUSSION THREAD

**Discussion by Invictus-Ronin** [GriefPrevention/GriefPrevention#696] [Q&A] [answered]
This commands (the ones without /) don't work in my server's console nor in-game. How can I use them to configure the plugin in my server?

**Comment by bigpresh** [comment, marked as answer]
... that's not a command, that's a config entry.

The commands are all documented at https://github.com/TechFortress/GriefPrevention/wiki/Commands

`Claims.PreventTheft: true` as per your issue title is a config directive it goes in your config.yml file.  

The config file format is documented at https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration
